test/backend/numpy.in.cpp

```diff
@@ -0,0 +1,116 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <algorithm>
+#include <cinttypes>
+#include <cmath>
+#include <cstdlib>
+#include <random>
+#include <string>
+
+// clang-format off
+#ifdef ${BACKEND_NAME}_FLOAT_TOLERANCE_BITS
+#define DEFAULT_FLOAT_TOLERANCE_BITS ${BACKEND_NAME}_FLOAT_TOLERANCE_BITS
+#endif
+
+#ifdef ${BACKEND_NAME}_DOUBLE_TOLERANCE_BITS
+#define DEFAULT_DOUBLE_TOLERANCE_BITS ${BACKEND_NAME}_DOUBLE_TOLERANCE_BITS
+#endif
+// clang-format on
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "util/all_close.hpp"
+#include "util/all_close_f.hpp"
+#include "util/ndarray.hpp"
+#include "util/test_control.hpp"
+#include "util/test_tools.hpp"
+
+#include <pybind11/embed.h>
+#include <pybind11/numpy.h>
+
+using namespace std;
+using namespace ngraph;
+
+namespace py = pybind11;
+using namespace py::literals;
+
+static string s_manifest = "${MANIFEST}";
+
+NGRAPH_TEST(${BACKEND_NAME}, numpy_abc)
+{
+    auto backend = runtime::Backend::create("${BACKEND_NAME}");
+
+    int a = 3, b = 4;
+
+    auto locals = py::dict("a"_a = a, "b"_a = b);
+    py::exec(R"(
+        c = a + b
+    )",
+             py::globals(),
+             locals);
+
+    auto sum = locals["c"].cast<int>();
+
+    EXPECT_TRUE(sum == a + b);
```

Comment:
Should "a+b" go through the backend? if not, then do we need the "backend" in line 55?

Reply:
Will remove line 55.

Reply:
Removed.